rootfs/etc/nginx/lua/metric_emitter.lua

```diff
@@ -0,0 +1,56 @@
+local socket = ngx.socket.udp
+local cjson = require('cjson')
+local assert = assert
+
+local _M = {
+  queue = {}
+}
+
+local function flush_queue()
+  s = assert(socket())
+  assert(s:setpeername("127.0.0.1", 8000))
+  for _, v in ipairs(_M.queue) do 
+    assert(s:send(v))
+  end
+  _M.queue = {}
+  assert(s:close())
+end
+
+function _M.call()
+  local current_phase = ngx.get_phase()
+  if current_phase == "log" then
```

Comment:
Could we get rid of the `current_phase` check?

This module could potentially be called from any phase given that it just defers the actual payload emission to the `timer` phase